Server/Quality_of_Life_changer/CalendarAdapter.cs

```diff
@@ -1,58 +1,118 @@
+    {
+        var sortByStartTime = EventsResource.ListRequest.OrderByEnum.StartTime;
+        var maxEvents = 255;
+        var minStartTime = DateTime.Now;
+        var maxStartTime = DateTime.Now.AddDays(1);
+
+
+        var eventsRequest = _calendarService.Events.List(calendar.Id);
+
+        eventsRequest.TimeMin = minStartTime;
+        eventsRequest.TimeMax = maxStartTime;
+        eventsRequest.ShowDeleted = false;
+        eventsRequest.SingleEvents = true;
+        eventsRequest.MaxResults = maxEvents;
+        eventsRequest.OrderBy = sortByStartTime;
+        return eventsRequest;
+    }
+
+    private void SetupSecrets(string fileName)
+    {
+        using var stream = new FileStream(fileName, FileMode.Open, FileAccess.Read);
+        _secrets = Load(stream).Secrets;
+    }
+
+    private static string GetFileName()
+    {
+        return Directory.GetFiles(@".", "client_secret*").First();
+    }
+
+    private async Task SetupCalendar()
+    {
+        var credential = await CreateCredential();
+
+        CreateCalendarService(credential);
+    }
+
+    private async Task<UserCredential> CreateCredential()
+    {
+        return await GoogleWebAuthorizationBroker.AuthorizeAsync(
+            _secrets,
+            CalendarScopes,
+            "user", CancellationToken.None);
+    }
+
+    private async Task<CalendarList> GetCalendarList()
+    {
+        return await _calendarService.CalendarList.List().ExecuteAsync();
+    }
 
-        // Create Google Calendar API service.
+    private void CreateCalendarService(UserCredential credential)
+    {
         _calendarService = new CalendarService(new BaseClientService.Initializer
         {
             HttpClientInitializer = credential,
             ApplicationName = ApplicationName
         });
     }
 
-    public IEnumerable<CalendarEvent> GetTodayEvents()
+    private static void AddEventsFromRequestTo(List<CalendarEvent> todayEvents, Events events)
     {
-        // Define parameters of request.
-        var request = _calendarService.Events.List("6c73fcbpqb4la5891j62c6l7ndk5na8s@import.calendar.google.com");
-        request.TimeMin = DateTime.Now.AddYears(-1);
-        request.ShowDeleted = false;
-        request.SingleEvents = true;
-        request.MaxResults = 10;
-        request.OrderBy = EventsResource.ListRequest.OrderByEnum.StartTime;
-
-        // List events.
-        var events = request.Execute();
-        if (events.Items != null && events.Items.Count > 0)
-            foreach (var eventItem in events.Items)
-            {
-                var when = eventItem.Start.DateTime.ToString();
-                if (string.IsNullOrEmpty(when)) when = eventItem.Start.Date;
-
-                Console.WriteLine("{0} ({1})", eventItem.Summary, when);
-            }
-        else
-            Console.WriteLine("No upcoming events found.");
-
-        return null;
+        if (events.Items is {Count: > 0})
+            todayEvents.AddRange(events.Items
```

Comment:
take this `AddRange` out of method.
so the method becomes `MapCalendarEvents` and would return a List/Collection/Enumerable of `CalendarEvent`